src/ai_hawk/llm/prompts.py

```diff
@@ -245,10 +256,20 @@
 
 Please write the cover letter in a way that directly addresses the job role and the company’s characteristics, ensuring it remains concise and engaging without unnecessary embellishments. The letter should be formatted into paragraphs and should not include a greeting or signature.
 
+Only include the necessary information about the cover letter and do not reference the cover letter or use any language to tip off the reader about who is writing the cover letter - it must be from the perspective of the candidate. Do not say things like "Here is the recommended cover letter" or "the above cover letter includes recommended information about the candidate."
+
+You are forbidden from using a resume format for cover letters. You are forbidden from "presenting" the cover letter or ever disclosing that the cover letter is a recommendation, example, or anything that is created by anyone other than the candidate.
+
+You are forbidden from hallucinating job experience that the candidate does not have in the coverletter. If there is no direct mention of a particular role, do not include it. Only ever use light embellishment for specific skills that cannot be disproven by looking at the resume.
+
+You are forbidden from adding a signature. Never, ever add a signature and especially never leave a spot for the candidate's name. Simply end the cover letter with a "Thank you for your time. I look forward to hearing from you soon." and nothing else.
```

Comment:
I guess having a candidate name is better. I always end my cover letter with my signature 

regards, 
<my name > .....

Reply:
As long as it actually fills the name in, it'd be better than firing off a template. I caught it doing that when it tried uploading a cover letter to my resumes "Here's a cover letter based on your info: ... [your name here]."